sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3249,6 +3249,11 @@ object GpuOverrides extends Logging {
       (c, conf, p, r) => new TypedImperativeAggExprMeta[ApproximatePercentile](c, conf, p, r) {
 
         override def tagAggForGpu(): Unit = {
```

Comment:
If you look at `Upper` or `Lower` as an example there is a built in way to mark an entire operator as incompat and it lets us put in a doc string with it to explain how it is incompat.

```
}).incompat("The GPU implementation of approx_percentile is not bit-for-bit " +
    "compatible with Apache Spark.")
```

Reply:
Ah, I didn't realize we had that. That looks much better and I will update this.